src/main/java/org/osc/sdk/manager/api/ApplianceManagerApi.java

```diff
@@ -16,128 +16,134 @@
      */
     ManagerPolicyApi createManagerPolicyApi(ApplianceManagerConnectorElement mc) throws Exception;
 
     /**
-     * @param mc
-     *            Manager Connector Element {@link ApplianceManagerConnectorElement}
-     * @return Domain Api. It can be null if {@code #isPolicyMappingSupported()} returns false.
-     * @throws Exception
+     * @param mc  provides information needed to connect with the security manager
+     * @return the APIs used by OSC to retrieve domains from the security managers
+     * @throws Exception when an instance of the API cannot be created
      */
     ManagerDomainApi createManagerDomainApi(ApplianceManagerConnectorElement mc) throws Exception;
 
     /**
-     * @param mc
-     *            Manager Connector Element {@link ApplianceManagerConnectorElement}
-     * @return Agent Api. It can be null if {@code #isAgentSupported()} returns false.
-     * @throws Exception
+     * @param mc  provides information needed to connect with the security manager
+     * @param vs  provides context information of the virtual system, see {@link VirtualSystemElement}
+     * @return the APIs used by OSC to manage device members
+     * @throws Exception when an instance of the API cannot be created
      */
     ManagerDeviceMemberApi createManagerDeviceMemberApi(ApplianceManagerConnectorElement mc, VirtualSystemElement vs) throws Exception;
 
     /**
+     * @param mc  provides information needed to connect with the security manager
+     * @return the APIs used by OSC to manage notification registration
      *
-     * Implementation of this method is required if Manager uses Web Socket Subscription Type for Notifications
-     *
-     * @param mc
-     *            Manager Connector Element {@link ApplianceManagerConnectorElement}
-     * @return Web Socket Notification Api
-     * @throws UnsupportedOperationException
-     *             if WebSocket Notification is not supported
-     * @throws Exception
+     * @throws UnsupportedOperationException if the notification type is
+     * not {@link ManagerNotificationSubscriptionType#TRANSIENT_WEB_SOCKET}
+     * @throws Exception when an instance of the API cannot be created
      */
     ManagerWebSocketNotificationApi createManagerWebSocketNotificationApi(ApplianceManagerConnectorElement mc)
             throws Exception;
 
     /**
-     * @param mc
-     *            Manager Connector Element {@link ApplianceManagerConnectorElement}
-     * @return Callback Notification Api
-     * @throws UnsupportedOperationException
-     *             if Callback Notification is not supported
-     * @throws Exception
+     * @param mc  provides information needed to connect with the security manager
+     * @return the APIs used by OSC to manage notification registration
+     *
+     * @throws UnsupportedOperationException if the notification type is
+     * not {@link ManagerNotificationSubscriptionType#CALLBACK_URL}
```

Comment:
In the javadoc it shows "notification type is not ManagerNotificationSubscriptionType.CALLBACK_URL" adding the suggested extra text will make it look redundant.